<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuerylogWindow</class>
 <widget class="QWidget" name="QuerylogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>362</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>362</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>362</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查询检测记录</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>查询计划名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="planName_combo">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>查询机组名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="machineName_combo">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="ok_btn">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_btn">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
